uc-crux-llvm/src/UCCrux/LLVM/Setup/Monad.hs

```diff
@@ -276,7 +277,7 @@ malloc ::
   -- | Path to this pointer
   Selector m argTypes inTy ('FTPtr atTy) ->
   -- | Size, as in number of elements. Should be strictly positive.
```

Comment:
```suggestion
  -- | Size, as in number of elements
```

Reply:
I think this comment still applies - we don't want to `malloc(0)`.